packages/tools/examples/videoSplineROITools/index.ts

```diff
@@ -0,0 +1,318 @@
+      const splineConfig = toolGroup.getToolConfiguration(
+        splineToolName,
+        'spline'
+      );
+
+      splineConfig.configuration[splineType].resolution = value;
+      toolGroup.setToolConfiguration(splineToolName, { spline: splineConfig });
+    });
+  },
+});
+
+addSliderToToolbar({
+  id: 'splineScale',
+  title: 'Scale',
+  range: [0, 1],
+  step: 0.1,
+  defaultValue: DEFAULT_CARDINAL_SCALE,
+  onSelectedValueChange: (value: string) => {
+    const toolGroup = ToolGroupManager.getToolGroup(toolGroupId);
+    const splineToolName = 'CardinalSplineROI';
+
+    const splineConfig = toolGroup.getToolConfiguration(
+      splineToolName,
+      'spline'
+    );
+
+    splineConfig.configuration.CARDINAL.scale = value;
+    toolGroup.setToolConfiguration(splineToolName, { spline: splineConfig });
+  },
+});
+
+/**
+ * Runs the demo
+ */
+async function run() {
+  // Init Cornerstone and related libraries
+  await initDemo();
+
+  // Add tools to Cornerstone3D
+  cornerstoneTools.addTool(SplineROITool);
+  cornerstoneTools.addTool(LivewireContourTool);
+
+  // Define a tool group, which defines how mouse events map to tool commands for
+  // Any viewport using the group
+  const toolGroup = ToolGroupManager.createToolGroup(toolGroupId);
+
+  toolGroup.addToolInstance('CardinalSplineROI', SplineROITool.toolName, {
+    spline: {
+      type: SplineROITool.SplineTypes.Cardinal,
+      configuration: {
+        [SplineROITool.SplineTypes.Cardinal]: {
+          scale: DEFAULT_CARDINAL_SCALE,
+        },
+      },
+    },
+  });
+
+  toolGroup.addToolInstance('CatmullRomSplineROI', SplineROITool.toolName, {
+    spline: {
+      type: SplineROITool.SplineTypes.CatmullRom,
+    },
+  });
+
+  toolGroup.addToolInstance('LinearSplineROI', SplineROITool.toolName, {
+    spline: {
+      type: SplineROITool.SplineTypes.Linear,
+    },
+  });
+
+  toolGroup.addToolInstance('BSplineROI', SplineROITool.toolName, {
+    spline: {
+      type: SplineROITool.SplineTypes.BSpline,
+    },
+  });
+
+  toolGroup.addTool(LivewireContourTool.toolName);
+
+  // Set the initial state of the tools, here we set one tool active on left click.
+  // This means left click will draw that tool.
+  toolGroup.setToolActive(toolsNames[0], {
+    bindings: [
+      {
+        mouseButton: MouseBindings.Primary, // Left Click
+      },
+    ],
+  });
+
+  addManipulationBindings(toolGroup);
+
+  // Get Cornerstone imageIds and fetch metadata into RAM
+  const imageIds = await createImageIdsAndCacheMetaData({
+    StudyInstanceUID: '2.25.96975534054447904995905761963464388233',
+    SeriesInstanceUID: '2.25.15054212212536476297201250326674987992',
+    wadoRsRoot:
+      getLocalUrl() || 'https://d33do7qe4w26qo.cloudfront.net/dicomweb',
+  });
+
+  // Only one SOP instances is DICOM, so find it
+  const videoId = imageIds.find(
+    (it) => it.indexOf('2.25.179478223177027022014772769075050874231') !== -1
+  );
```

Comment:
Can we please add a section to here https://www.cornerstonejs.org/docs/concepts/cornerstone-core/viewports to explain we have a video viewport and how to set an actual video there?

And ideally a new tutorial https://www.cornerstonejs.org/docs/category/tutorials

We have budget left :)

Reply:
Already have the second one: https://www.cornerstonejs.org/docs/tutorials/basic-video
Adding the first one.